src/Logger.js

```diff
@@ -62,81 +63,109 @@ export default class Logger {
       return acc;
     }, {});
 
-    this.winston = new winston.Logger({
+    this.winston = winston.createLogger({
       transports: [
         new winston.transports.Console({
-          stderrLevels: this.stderrLevels,
-          handleExceptions: false,
-          colorize: this.colorize,
-          formatter: this.isProduction
-            ? (...args) => this.formatJSONMessage(...args)
-            : undefined,
-          level: this.level
+          stderrLevels: this.stderrLevels
         })
       ],
+      format: this.format(),
+      level: this.level,
       levels: this.winstonLevels,
-      colors: this.colors
+      handleExceptions: false,
+      exitOnError: this.exitOnError
     });
-
-    // winston doesn't allow to set the error level for runtime
-    // exceptions, therefore this should be handled outside of winston.
-    // https://github.com/winstonjs/winston/blob/bd06c4091e9e248e6fed6278e0699e5cfdc7de24/lib/winston/exception-handler.js#L94-L97
-    process.on('uncaughtException', this.onUnhandledException);
-
-    // Unhandled promise rejections should be treated as errors.
-    process.on('unhandledRejection', this.onUnhandledException);
```

Comment:
Great that this is gone! Can you add "resolves #19" to the PR description to close https://github.com/molindo/molindo-node-logger/issues/19 when this is merged?